.travis.yml

```diff
@@ -6,11 +6,9 @@ script: "bundle exec rake validate && bundle exec rake lint && bundle exec rake
 matrix:
   fast_finish: true
   include:
-  - rvm: 1.8.7
-    env: PUPPET_GEM_VERSION="~> 3.0"
   - rvm: 1.9.3
     env: PUPPET_GEM_VERSION="~> 3.0"
-  - rvm: 2.0.0
```

Comment:
Well, I would strongly encourage you to keep support for 1.8.7 as it is what ships with EL6. Though if you want to remove it, I would also remove it from the README and do so in its own release that is separate from this PR, so it is clear that at release X, ruby 1.8.7 was dropped.

I don't personally use 2.0.0 and am not sure which OS's default to using it, though I have support for it in all of my modules. Would probably keep that around as it is newer.